tests/integration/stop_consumer.go

```diff
@@ -80,33 +79,6 @@ func (s *CCVTestSuite) TestStopConsumerChain() {
 				return nil
 			},
 		},
-		{
-			func(suite *CCVTestSuite) error {
```

Comment:
Any test or test case that was removed, is a test that is only relevant to provider queuing logic which is removed in this PR